src/reducers/controlFieldUtils.js

```diff
@@ -93,3 +95,51 @@ export function fillControlFieldGaps(baseField, sourceField, min = 0, max = 39)
 
   baseField.value = mergedCharArray.join(''); // eslint-disable-line functional/immutable-data
 }
+
+export function genericControlFieldCharPosFix(baseField, sourceField, typeOfMaterial, supportedTypesOfMaterial, legalValues, position, valueForUnknown, noAttemptToCode) { // eslint-disable-line max-params
```

Comment:
Give parameters as object to avoid max-params problem?

Reply:
Modified this:  rule is used instead of former supportedTypesOfMaterial, legalValues, position, valueForUnknown, and noAttemptToCode. The current solution still contains 5 parameters (of which one is actually a new one). I don't want to use objects just to get rid of this warning